data/members/Annabelle Feiler.json

```
@@ -0,0 +1,14 @@
+{
+  "date": "2024-01-09T00:00",
+  "name": "Annabelle Feiler",
+  "url": "https://annabelle-feiler.com/",
+  "employment": {
+    "hiring": false,
+    "seeking": false
+  },
+  "github": null,
+  "linkedin": "https://www.linkedin.com/in/annabelle-feiler/",
+  "mastodon": null,
+  "rss": null,
+  "twitter": null,
```

```suggestion
  "twitter": null
```

Remove trailing comma.